modules/primer-markdown/stories.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
```

Comment:
🎉 Yay for adding more stories! It would be great to add the rest of the markdown styles when you get a chance.